### ISSUE TITLE
Look into managing multiple databases from one install

### ISSUE THREAD

**Issue by benhaynes** [directus/directus#982]
I assume this would be a global dropdown to change the DB you're connected to.

**Comment by hrwebasst** [comment]
I'd suggest moving this further up the issue list. I have a use case that is probably pretty common. Installing automatically in my docker instance with my site download for development purposes. I have already easily bypassed the login and can add the schema in by default but it would be nice if Directus could be simply pointed at a database the user had perms for so that I can bake this for all of my users.

*tl;dr* Directus has own database points to one or multiple sub-databases on install would be awesome.

**Comment by benhaynes** [comment]
**[Migrated to FeatHub](http://feathub.com/directus/directus/+22)**

Please use the [Directus FeatHub](http://feathub.com/directus/directus) to vote for features or make new request. Accepted feature requests will be added to GitHub Issues.

@hrwebasst – feel free to vote this one to the top over on FeatHub!
